masterlist.yaml

```
@@ -15485,12 +15491,19 @@ plugins:
         name: 'Relationship Dialogue Overhaul - RDO SE'
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/42068/'
         name: 'Relationship Dialogue Overhaul Lite Fixed'
-    after: [ '018Auri.esp' ]
+    after:
+      - '018Auri.esp'
+      - 'alternate start - live another life.esp'
```

```suggestion
    after: [ '018Auri.esp' ]
```

This rule has come up frequently over the years (#576, #715, #1311) & was removed in #1312 since it doesn't work for every load order. Not much has changed since, so we should still leave it to user preference.